app/views/content/landing/grow/_mailing_list.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <section class="col col-720">
+    <%= render Content::MailingListComponent.new(
+      title: "Find out about teacher training",
+      intro: "Get helpful tips and advice on getting into teacher training straight to your inbox by answering just a few quick questions about yourself."
```

Comment:
@MaxineCl Should we make this more engaging? Linked to the campaign?

Reply:
@gemmadallmandfe I've updated the title and intro.